diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index 6348d17..7da0ed1 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -42,7 +42,7 @@ def resolveReactNativeDirectory() {
     def reactNativePackage = file(
         providers.exec {
             workingDir(rootDir)
-            commandLine("node", "--print", "require.resolve('react-native/package.json')")
+            commandLine("/Users/sridixtech/.nvm/versions/node/v24.11.1/bin/node", "--print", "require.resolve('react-native/package.json')")
         }.standardOutput.asText.get().trim()
     )
 
